### PR TITLE
Add location to v1 nodes

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

add location object to `v1/nodes`

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: sensorsafrica/api/v1/serializers.py [hunk 1]
```diff
@@ -10,10 +10,13 @@
     NestedSensorSerializer,
     SensorDataSerializer as PostSensorDataSerializer
 )
+class NodesLocationSerializer(NestedSensorLocationSerializer):
```
Comment (kilemensi) [inline sensorsafrica/api/v1/serializers.py:13]:
```suggestion
class NodeLocationSerializer(NestedSensorLocationSerializer):
```

**Review by kilemensi — APPROVED** [review]
LGTM @KhadijaMahanga

**Comment by esirK** [comment]
@kilemensi  should we be merging to develop or master?

**Comment by kilemensi** [comment]
> @kilemensi should we be merging to develop or master?

Didn't you merge `develop` to `master` and hopeful delete `develop` yesterday @esirK ?

**Comment by esirK** [comment]
@kilemensi I did merge develop into master but didn't delete it. Updating this PR to point the merge into master.

**Comment by KhadijaMahanga** [comment]
@esirK  why did you change back to `develop`?

**Comment by esirK** [comment]
> @esirK why did you change back to `develop`?

The number of commits. I'm not sure it's correct

**Comment by kilemensi** [comment]
If it's an issue @KhadijaMahanga can you push to a different branch and we then delete both this and `develop` branches?

**Pull request closed without merging**